.gitlab-ci.yml

```diff
@@ -2,26 +2,30 @@ image: node:10
 
 variables:
   NODE_ENV: ci
+  NEARCORE_DIR: nearcore
 
 stages:
   - build
   - test
 
 cache:
   paths:
+    - nearcore/
     - node_modules/
 
 install_dependencies:
   stage: build
   script:
+    - rm -rf nearcore/
```

Comment:
could you leave a comment why this is needed? It's definitely not intuitive.

Reply:
probably makes more sense to remove `nearcore` from cache, then this won't be needed

Reply:
added comment